packages/server/lib/util/ci_provider.js

```diff
@@ -505,6 +516,10 @@ const _providerCommitParams = () => {
       // defaultBranch: ???
     },
     wercker: null,
+    netlify: {
+      sha: env.COMMIT_REF,
+      branch: env.BRANCH,
```

Comment:
They're sending repo url too, we can use this. 

```suggestion
      branch: env.BRANCH,
      remoteOrigin: env.REPOSITORY_URL
```